src/main/java/uk/gov/hmcts/reform/sscs/transform/deserialize/SubmitYourAppealToCcdCaseDataDeserializer.java

```diff
@@ -38,6 +38,7 @@ public static SscsCaseData convertSyaToCcdCaseData(SyaCaseWrapper syaCaseWrapper
             .toBuilder()
             .region(region)
             .regionalProcessingCenter(rpc)
+                .schedulingAndListingFields(SchedulingAndListingFields.builder().hearingRoute(rpc.getHearingRoute()).build())
```

Comment:
don't think this should be indented, as it implies schedulingAndListingFields are part of regionalProcessingCenter, which they're not

Reply:
is there a unit test for this?

Reply:
Yes, in SYA tests there are checks that cover if the page has specific fields by using the json file - so i added hearingRoute to this file, so that it checks whether this field is present.